.travis.yml

```diff
@@ -29,6 +29,10 @@ matrix:
       - php: 7.2
         env: BUZZ_TEST_SERVER=""  CS=true
 
+      - php: 7.2
+        name: Force HTTPlug 2.0
+        env: DEPENDENCIES="php-http/httplug:2.0.*"
```

Comment:
as 2.0 is a highest version that 1.x, that's useless. The normal job will already be using it on supported PHP versions.